mlflow/tracing/utils.py

```diff
@@ -80,3 +80,31 @@ def format_trace_id(trace_id: int) -> str:
     Format the given integer trace ID to a hex string.
     """
     return f"0x{trace_api.format_span_id(trace_id)}"
+
+
+def decode_span_id(span_id: str) -> int:
+    """
+    Decode the given hex string span ID to an integer.
+    """
+    return int(span_id, 16)
+
+
+def decode_trace_id(trace_id: str) -> int:
+    """
+    Decode the given hex string trace ID to an integer.
+    """
+    return int(trace_id, 16)
```

Comment:
Can we unify these two if they're the same?

Reply:
Ooh they shouldn't be same (trace_id is encoded to 32 hex not 16). The reason it was not caught in test (and copilot substitute 16 here for both) is I mistakenly put `trace_api.format_span_id` for both L74 and L82... Thanks for the catch!